File: Java/src/main/java/LCP/Solution6.java
package LCP;
/**
 * @author windows
 */
public class Solution6 {
    public int minCount(int[] coins) {
        int sum = 0;
        for (int i = 0; i < coins.length; i++) {
            sum += coins[i] / 2;
            sum += coins[i] % 2;
        }
        return sum;
    }
    
    
    public static void main(String[] args) {
        Solution6 solution6 = new Solution6();
    }
}

File: C++/src/LCR/Solution6.cpp
//
// Created by windows on 2023/12/19.
//

#include "Solution6.h"
#include <functional>
#include <vector>
using namespace std;
using namespace std;

int minCount(vector<int> &coins) {
  int count = 0;
  for (const auto &item : coins) {
    count += item / 2 + item % 2;
  }
  return count;
}
